platforms/hyperledger-besu/configuration/roles/create/helm_component/templates/crypto_tessera.tpl

```diff
@@ -25,6 +25,7 @@ spec:
       repository: quorumengineering/tessera:hashicorp-{{ network.config.tm_version }}
       pullSecret: regcred
       pullPolicy: IfNotPresent
+      alpineutils: {{ network.docker.url }}/alpine-utils:1.0
```

Comment:
Lets not call this `v1.0` . You can use `v1.1` for this as we might update this image in the very near future for next major release of Bevel.

Reply:
Updated the image version to v1.1